client/application.html

```diff
@@ -7,31 +7,37 @@
 
 		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/wingcss/0.1.8/wing.min.css" />
 		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,300,700" />
+		<link rel="stylesheet" href="/node_modules/sweetalert2/dist/sweetalert2.min.css" />
 		<link rel="stylesheet" href="/css/main.css" />
 		<link rel="stylesheet" href="/css/application.css" />
-		<script src="https://use.fontawesome.com/efaca8fe24.js"></script>
 		<script src="https://cdnjs.cloudflare.com/ajax/libs/qwest/4.4.5/qwest.min.js"></script>
+		<script src="/node_modules/sweetalert2/dist/sweetalert2.min.js"></script>
+		<script src="/js/application.js" defer></script>
 	</head>
 	<body>
 		{{#> sidebar}}
 			<h1 class="center">Apply</h1>
-			
-			<form method="POST" action="/apply">
+
+			<!--
+				Set data-action and not action so that verification of the form still occurs but the form is actually submitted via an XHR.
+				Running event.preventDefault() in the submit button's click handler also disables validation for some reason
+			-->
+			<form method="post" data-action="/api/user/{{user._id}}/application">
```

Comment:
The template is currently compiled server-side at startup ([see here](https://github.com/HackGT/registration/blob/application-submission/server/routes/templates.ts#L25)) and then the parameters are inserted into every request if that makes any sense ([see here](https://github.com/HackGT/registration/blob/application-submission/server/routes/templates.ts#L103)).